openshift/helpers.sh

```diff
@@ -0,0 +1,116 @@
+
+function is_set_or_fail() {
+    local name=$1
+    local value=$2
+
+    if [ -v $value ] || [ $value == "not-set" ]; then
```

Comment:
```
    if [ -v $value ] || [ $value == "not-set" ]; then
            ^-- SC2086: Double quote to prevent globbing and word splitting.
                          ^-- SC2086: Double quote to prevent globbing and word splitting.
```